.github/workflows/main.yml

```diff
@@ -19,53 +21,23 @@ jobs:
     - name: setup-msbuild
       uses: microsoft/setup-msbuild@v1
 
-    - name: Restore
-      run: msbuild /restore Xamarin.Forms.sln
-    
-    - name: Build
-      run: msbuild .\Xamarin.Forms.sln -p:Configuration=Release
+    - name: Restore & Build
```

Comment:
@parhamsaremi why are you merging the restore and build steps? I never asked for this

Reply:
Afshin used merged steps. So I figured I should do the same.

Reply:
Steps should be as small as possible, so that when they fail, you know exactly what thing failed. If Afshin had fat steps, it's my fault for not pointing out in the review.

Reply:
Ummm sure, I'll apply your comments now.